<?xml version="1.0"?>
<robot name="push_box">
  <link name="push_box">
    <visual>
      <origin xyz="0.0 0.0 0.0"/>
      <geometry>
        <mesh filename="/home/dvrk/catkin_ws/src/dvrk_env/shape_servo_control/src/teleoperation/random_stuff/new_push_box.obj" scale="1.0 1.0 1.0"/>
      </geometry>
    </visual>
    <collision>
      <origin xyz="0.0 0.0 0.0"/>
      <geometry>
        <mesh filename="/home/dvrk/catkin_ws/src/dvrk_env/shape_servo_control/src/teleoperation/random_stuff/new_push_box.obj" scale="1.0 1.0 1.0"/>
      </geometry>
    </collision>
    <inertial>
      <mass value="1"/> #0.01
      <inertia ixx="1" ixy="1" ixz="1" iyy="1" iyz="1" izz="1"/>
    </inertial>
  </link>
</robot>
    

<!--scale="0.003 0.003 0.003"for kindney 2-->
